### ISSUE TITLE
Quick snippet didn't work well when OB's default language is non-EN 

### ISSUE THREAD

**Issue by Quorafind** [liamcain/obsidian-calendar-plugin#71]
I have adopted new features({{sunday:YYYYMMDD}}) in my **weekly note** template. 
Then I want to get my whole week's every day note by using:

```
![[{{sunday:YYYYMMDD}} DailyNote]]
![[{{monday:YYYYMMDD}} DailyNote]]
![[{{tuesday:YYYYMMDD}} DailyNote]]
![[{{wednesday:YYYYMMDD}} DailyNote]]
![[{{thursday:YYYYMMDD}} DailyNote]]
![[{{friday:YYYYMMDD}} DailyNote]]
![[{{saturday:YYYYMMDD}} DailyNote]]
```
Its output (Ob's default language is Simplified-Chinese) after weekly note generated:

```
![[20201205 DailyNote]]
![[20201128 DailyNote]]
![[20201121 DailyNote]]
![[20201114 DailyNote]]
![[20201107 DailyNote]]
![[20201031 DailyNote]]
![[20201024 DailyNote]]
```

I also try it with another non-EN language as default language. It seems the same problem. But it worked well when I use EN as default lanuage.
Then I also tried it in my new vault with any notes in it, things didn't change.

**Comment by liamcain** [comment]
I see, thanks for finding this! I see what the issue is. Will make a fix in the next day or 2.

**Comment by liamcain** [comment]
Sorry for the delay, this is now fixed in v1.4.5. Let me know if you're still having issues.
